test/e2e/apps/types.go

```diff
@@ -35,4 +35,7 @@ var (
 
 	// AgnhostImage is the fully qualified URI to the Agnhost image
 	AgnhostImage = imageutils.GetE2EImage(imageutils.Agnhost)
+
+	// InvalidImage is a fully qualified URI pointing to an non-existent image
+	InvalidImage = imageutils.GetE2EImage(imageutils.InvalidRegistryImage)
```

Comment:
unrelated to this PR but I just realized that we are sending requests to the domain `invalid.com`

https://github.com/kubernetes/kubernetes/blob/c82a0f8ddcae7b53b72adca8e492feca20396e44/test/utils/image/manifest.go#L93

registered by 

https://whois.domaintools.com/invalid.com

I think we should try to avoid to create requests to an uncontrolled domain, IMHO, we should add a fake domain that the project controls `invalid.registry.k8s.io` per example

/cc @dims @BenTheElder @thockin

Reply:
PR to create a dummy record `invalid.registry.k8s.io` https://github.com/kubernetes/k8s.io/pull/3245